library/src/main/java/com/yausername/youtubedl_android/DownloadProgressCallback.java

```diff
@@ -1,5 +1,5 @@
 package com.yausername.youtubedl_android;
 
 public interface DownloadProgressCallback {
-    void onProgressUpdate(float progress, long etaInSeconds, String line);
+    void onProgressUpdate(String line);
```

Comment:
Can we do this change without breaking the API?
I mean keep the same method signature as before.
We can use some reasonable defaults for `progress` and `etaInSeconds` when the line does not match the download pattern. For example 0, -1 before the download starts and 100, 0 after it is completed.

Reply:
Done, maybe it's better now.